src/libostree/ostree-bootconfig-parser.h

```diff
@@ -67,4 +67,13 @@ void ostree_bootconfig_parser_set_overlay_initrds (OstreeBootconfigParser *self,
 _OSTREE_PUBLIC
 char **ostree_bootconfig_parser_get_overlay_initrds (OstreeBootconfigParser *self);
 
+_OSTREE_PUBLIC
+guint64 ostree_bootconfig_parser_get_tries_left (OstreeBootconfigParser *self);
+
+_OSTREE_PUBLIC
```

Comment:
Why are these public APIs? I'm not saying it's wrong, but it's not immediately obvious to me why they should be.

Well, I guess actually we probably should be rendering this in e.g. `ostree admin status` at least.

But right now only `_tries_left` is listed in the symbols file, so if we're making these public we should ensure that `_tries_done` is too.